DisCatSharp.Docs/articles/basics/web_app.md

```diff
@@ -96,13 +107,10 @@ IBot bot = provider.GetRequiredService<IBot>();
 If you go down this path of mapping interface to implementation you shouldn't be casting your interface to Bot, or whatever. You'd be better off just using the explicitly registered type.
 The reasoning behind this approach is to allow you to swap out the implementation type in **ONE** place, and **NOT** have to update any other code.
 
-For instance, logging... there are SO many ways to do logging. You might notice, or be familiar with `ILogger`. So long as something implements this interface it doesn't matter. It could be Serilog,
+For instance, logging... there are SO many ways to do logging. You might be familiar with `ILogger`. So long as something implements this interface it doesn't matter. It could be Serilog,
```

Comment:
```suggestion
For instance, logging... there are SO many ways to do logging. You might be familiar with [ILogger](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.ilogger). So long as something implements this interface it doesn't matter. It could be Serilog,
```